src/sections/Questionnaire/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import "./styles.css";
```

Comment:
Pass this as a component, why is this render itself?

Reply:
@tripathyprateek does this still need to be addressed?

Reply:
Yes @leecalcote . I didnt have a very good grasp of Gatsby Js at the time of making this PR, and made few mistakes in the process. I will have them addressed and make a better one.

Reply:
Oh, very good. 👍 No sweat. 😅